src/vs/editor/contrib/parameterHints/browser/parameterHints.ts

```diff
@@ -20,12 +20,12 @@ import { IInstantiationService } from 'vs/platform/instantiation/common/instanti
 import { KeybindingWeight } from 'vs/platform/keybinding/common/keybindingsRegistry';
 import { ParameterHintsWidget } from './parameterHintsWidget';
 
-class ParameterHintsController extends Disposable implements IEditorContribution {
+export const ParameterHintsContributionID = 'editor.controller.parameterHints';
```

Comment:
it would be more consistent with other contributions to just export the class and use the static ID member.

Reply:
Fixed, I was getting errors about import structure but no longer. It must have been due to something else.